src/app/Helpers/Import.php

```diff
@@ -40,8 +41,9 @@ class Import
         "Scraper Status" => -1,
     ];
 
-    public static function getLatestScrapedFile($prefix, $all_states = false) {
-        $latest_file = collect(Storage::disk('s3')->files())
+    public static function getLatestImportFile($prefix, $directory = '', $all_states = false) {
```

Comment:
If I understand the semantics of this function correctly, could we rename it to something like `getLastestVersionOfFile`? This function appears to take the _base_ name of a file and return the path to the latest version of it. If so, the updated name seems to make more sense to me. 

No requirement to change, just a suggestion!